File: ng-client/src/app/clothes.ts
export class Clothes {
  constructor(public id: string,
              public preview: string,
              public photo: string,
              public cost: number,
              public brand: string,
              public name: string,
              public size: string[],
              public color: string[],
              public published: Date) {
  }
}

File: moda-resource-server/src/main/java/com/moda/resource/persistence/model/Clothes.java
package com.moda.resource.persistence.model;

import lombok.Data;
import org.springframework.data.annotation.Id;

import java.util.Date;

@Data
public class Clothes {
    @Id
    private String id;
    private String category;
    private String brand;
    private String name;
    private Integer cost;
    private String[] color;
    private String[] size;
    private String photo;
    private String preview;
    private Date published;
}
